cylc/flow/suite_files.py

```diff
@@ -21,22 +21,30 @@
 
 import os
 from pathlib import Path
+from random import shuffle
 import re
 import shutil
+from subprocess import Popen, PIPE, DEVNULL
```

Comment:
Just noticed this line at the top:
```
# Note: Some modules are NOT imported in the header. Expensive modules are only
# imported on demand.
```
Should I import `Popen` etc inline instead? (This was done in `detect_old_contact_file()` before this)

Reply:
Eh, I think we can stop doing that and import at the top of the file like normal code.